python-sdk/nuscenes/eval/lidarseg/README.md

```diff
@@ -82,10 +83,16 @@ The contents of the `submision.json` file and `v1.0-test` folder are defined bel
   ```
 * The `v1.0-test` folder contains .bin files, where each .bin file contains the labels of the points for the point cloud.
   Pay special attention that each set of predictions in the folder must be a .bin file and named as **<lidar_sample_data_token>_lidarseg.bin**.
-  A .bin file contains an array of `int` in which each value is the predicted [class index](#classes) of the corresponding point in the point cloud, e.g.:
+  A .bin file contains an array of `uint8` values in which each value is the predicted [class index](#classes) of the corresponding point in the point cloud, e.g.:
   ```
   [1, 5, 4, 1, ...]
   ```
+  Below is an example of how to save the predictions for a single point cloud:
+  ```
+  bin_file_path = lidar_sample_data_token + '_lidarseg.bin"
+  np.array(predicted_labels).astype(np.uint8).tofile(bin_file_path)
+  ```
+  Note that the arrays should **not** contain the `ignore` class (i.e. class index 0). 
```

Comment:
if they do is it considered a failed submission as it will throw an assert?

Reply:
Yup! We are doing an assertion [here](https://github.com/nutonomy/nuscenes-devkit/pull/480/files#diff-2bf32c586a639b902970f73feb3b8b622a2f7dd3cdf506493ff4f97a3d6524e7R46)

Reply:
But no, an erroneous submission will not count towards the limit of 3 submissions.